Nos/Models/FlagOption.swift

```diff
@@ -67,19 +63,86 @@ struct FlagOption: Identifiable, Equatable {
         )
     ]
 
+    /// `FlagOption` instances representing different categories of the visibility of a flagged account.
+    static let flagAccountVisibilityCategories: [FlagOption] = [
```

Comment:
Similar to the above comment, maybe this?
```suggestion
    static let accountVisibilityOptions: [FlagOption] = [
```